library/maven/artifacts.bzl

```diff
@@ -227,6 +228,28 @@ artifacts = {
     "org.hamcrest:hamcrest-core": "1.3",
     "org.hamcrest:hamcrest-library": "1.3",
     "org.javatuples:javatuples": "1.2",
+    "org.jetbrains.compose.animation:animation-core-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.animation:animation-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.desktop:desktop-jvm": "1.0.0-alpha3",
+    "org.jetbrains.compose.foundation:foundation-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.foundation:foundation-layout-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.material:material-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.material:material-icons-core-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.material:material-ripple-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.runtime:runtime-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.runtime:runtime-saveable-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.ui:ui-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.ui:ui-geometry-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.ui:ui-graphics-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.ui:ui-text-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.ui:ui-tooling-preview-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.ui:ui-unit-desktop": "1.0.0-alpha3",
+    "org.jetbrains.compose.ui:ui-util-desktop": "1.0.0-alpha3",
+    "org.jetbrains.skiko:skiko-jvm": "0.3.9",
+    "org.jetbrains.skiko:skiko-jvm-runtime-linux-x64": "0.3.9",
+    "org.jetbrains.skiko:skiko-jvm-runtime-macos-x64": "0.3.9",
+    "org.jetbrains.skiko:skiko-jvm-runtime-windows-x64": "0.3.9",
+    "org.jetbrains:annotations": "13.0",
```

Comment:
I am not a fan of this massive dependency list, but removing them does seem to cause issues: see https://github.com/vaticle/typedb-workbase/issues/453

Reply:
I think `"org.jetbrains:annotations": "13.0"` should be at the top of the list of the newly added deps, since we want to keep the file in alphabetical / lexicographical order.